src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -57,7 +57,7 @@
     {%- endif -%}
     <select {{ block('widget_attributes') }}{% if multiple %} multiple="multiple"{% endif %}>
         {%- if empty_value is not none -%}
-            <option value=""{% if required and value is empty %} selected="selected"{% endif %}>{{ empty_value|trans({}, translation_domain) }}</option>
+            <option value=""{% if required and value is empty %} selected="selected"{% endif %}>{{ empty_value is empty ? empty_value : empty_value|trans({}, translation_domain) }}</option>
```

Comment:
when empty_value is empty, there is no point it outputting it. so you can do `{{ empty_value is not empty ? empty_value|trans({}, translation_domain) }}`